src/ctia/bundle/routes.clj

```diff
@@ -71,65 +72,71 @@
  (routes 
   (context "/bundle" []
            :tags ["Bundle"]
-           (GET "/export" []
-                :return NewBundleExport
-                :query [q BundleExportQuery]
-                :summary "Export records with their local relationships. Ids are URIs (with port if precised)."
-                :capabilities export-capabilities
-                :auth-identity identity
-                :identity-map identity-map
-                (ok (export-bundle
-                     (:ids q)
-                     identity-map
-                     identity
-                     q
-                     services)))
+           (let [capabilities export-capabilities]
+             (GET "/export" []
+                  :return NewBundleExport
+                  :query [q BundleExportQuery]
+                  :summary "Export records with their local relationships. Ids are URIs (with port if specified)."
+                  :description (common/capabilities->description capabilities)
```

Comment:
So I think this could be problematic, we should probably use something like this: 

https://github.com/threatgrid/iroh/blob/master/lib/iroh-web/src/iroh_web/compojure_api.clj#L25

Because we probably want to be able to prepend the capabilities to existing description.

Reply:
> So I think this could be problematic, we should probably use something like this:

I don't think that's possible because `:capabilities` is runtime information in CTIA. `-meta/restructure-param :scopes` takes compile-time information.

Reply:
Another piece of the puzzle is that the swagger metas can't see the `:lets` in scope in the same `restructure-param` call https://github.com/metosin/compojure-api/blob/1.1.13/src/compojure/api/meta.clj#L321